lib/content-services/upload/components/file-uploading-dialog.component.html

```diff
@@ -7,6 +7,8 @@
             mat-button
             color="secondary"
             [disabled]="isConfirmation"
+            [attr.aria-disabled]="isConfirmation"
```

Comment:
There's already a "disabled" attribute above. According to this discussion https://stackoverflow.com/a/38059790/1716421:
```
In short, you do not need aria-disabled any longer. Just use disabled.
```

Can you check if that is correct?

Reply:
yea, disabled makes the element not focusable so I guess we don't need it